Web Experiences/Blueprints/email-capture-corner/email-capture-corner.js

```diff
@@ -1,47 +1,83 @@
+//make experience unique
+var compiledCSS = Boxever.templating.compile(variant.assets.css)(variant);
+var styleTag = document.getElementById('style-' + variant.ref);
+if (styleTag) {
+    styleTag.innerHTML = compiledCSS;
+}
+/////
+
 insertHTMLAfter("body");
-var bxContent = document.querySelector("#bx-"+variant.ref+ " #bx-transition-card");
-bxContent.style.display = "flex";
-setTimeout(function(){
-    bxContent.classList.add("open");
-}, 10);
 
+// show popup on bx load
+let bxContent = document.querySelector("#bx-transition-card");
+bxContent.style.display = "flex";
 
+//declarations
 const bxEmailCaptureContainer = document.getElementById("bx-email_capture_container")
 const bxThankYouContainer = document.getElementById("bx-thank_you_container")
-const bxClose = bxContent.querySelector(".bx__btn-close-icon");
+const bxClose = bxContent.querySelector(".bx__btn-close__icon");
+const bxCTA = document.getElementById('bx-transition-card--primary');
+
 
-const bxButtonPress = document.getElementById('bx-transition-card--primary');
-function sendDataToBoxever(eventType) {
-    var eventToSend = {
+// LIsteners
+//on Email submission
+bxCTA.onclick = ()=>{
+    let bxEmail = document.getElementById("bx-email_input").value;
+    let emailVerified = validateEmail(bxEmail);
+    emailVerified ?
+        onSuccessValidation(bxEmail)
+    :
+        //friendly error
+        document.getElementById("bx-email_input").style.backgroundColor = 'rgba(200,0,0,0.1)';
+};
+
+bxClose.onclick = ()=>{
+    bxContent.style.display = "none";
+    sendDataToBoxever("INTERACTION_DISMISSED")
+}
+
+
+//declare functions
+const sendDataToBoxever = (eventType)=>{
+    let eventToSend = {
         "channel": "WEB",
         "type": eventType,
         "pos": window._boxever_settings.pointOfSale,
         "browser_id": Boxever.getID(),
         "interactionID":"OOB_EXP",
 	    "interactionName": "CORNER_POPUP_EMAIL"
     };
-    Boxever.eventCreate(eventToSend, function(data) {}, 'json');
+    Boxever.eventCreate(eventToSend, (data)=>{}, 'json');
 }
 
-bxButtonPress.onclick = function() {
+const onSuccessValidation = (email)=>{
     sendDataToBoxever("INTERACTION_IDENTITY")
-    var event = {
+    let event = {
         "channel": "WEB",
         "type": "IDENTITY",
         "language": "EN",
         "currency": "EUR",
         "page": "Home",
         "pos": "spinair.com",
         "browser_id": Boxever.getID(),
-        "email": document.getElementById("bx-email_input").value
+        "email":email
     };
-    Boxever.eventCreate(event, function(data) {}, 'json');
+    Boxever.eventCreate(event, (data)=>{}, 'json');
```

Comment:
This is incorrect here. Corrected Identity event:
```
let identityEvent = {
        "channel": "WEB",
        "type": "IDENTITY",
        "pos": window._boxever_settings.pointOfSale,
        "browser_id": Boxever.getID(),
        "email": email
    };
    Boxever.eventCreate(identityEvent, function(data) {}, 'json');

Reply:
Done